src/components/ClassCard.jsx

```diff
@@ -0,0 +1,257 @@
+          <StyledButtonWrapper>
+            <Buttons
+              buttonText="Book Now"
+              fontSize={15}
+              paddinglr={windowWidth >= 1024 ? 32 : 16}
+              width={windowWidth >= 744 ? 70 : windowWidth >= 448 ? 100 : 95}
+              className="class-button"
+            />
+          </StyledButtonWrapper>
+        </StyledTextButton>
+      </StyledCard>
+
+      {/* Second Card */}
+      <StyledCard>
+        <StyledClassImage src="/class02.png" alt="Get on the Mat" />
+        <StyledTextButton>
+          <StyledClassName>
+            <span className="class-keyword">Get on the Mat</span> - Learn the
+            fundamentals at your own pace.
+          </StyledClassName>
+          <StyledClassDescription>
+            A great place to start your pilates journey.
+          </StyledClassDescription>
+          <StyledButtonWrapper>
+            <Buttons
+              buttonText="Book Now"
+              fontSize={15}
+              paddinglr={windowWidth >= 1024 ? 32 : 16}
+              width={windowWidth >= 744 ? 70 : windowWidth >= 448 ? 100 : 95}
+              className="class-button"
+            />
+          </StyledButtonWrapper>
+        </StyledTextButton>
+      </StyledCard>
+
+      {/* Third Card */}
+      <StyledCard>
+        <StyledClassImage src="/class03.png" alt="Pre and Postnatal" />
+        <StyledTextButton>
+          <StyledClassName>
+            <span className="class-keyword">Pre and Postnatal</span>
+          </StyledClassName>
+          <StyledClassDescription>
+            Gentle exercises to maintain strength, flexibility, and promote
+            relaxation.
+          </StyledClassDescription>
+          <StyledButtonWrapper>
+            <Buttons
+              buttonText="Book Now"
+              fontSize={15}
+              paddinglr={windowWidth >= 1024 ? 48 : 16}
+              width={windowWidth >= 744 ? 70 : windowWidth >= 448 ? 100 : 95}
+              className="class-button"
+            />
+          </StyledButtonWrapper>
+        </StyledTextButton>
+      </StyledCard>
+
+      {/* Fourth Card */}
+      <StyledCard>
+        <StyledClassImage src="/class04.png" alt="On Demand" />
+        <StyledTextButton>
+          <StyledClassName>
+            <span className="class-keyword">On Demand</span> - workout wherever,
+            whenever, you decide.
+          </StyledClassName>
+          <StyledClassDescription>
+            Live classes online and over 100 pre-recorded sessions.
+          </StyledClassDescription>
+          <StyledButtonWrapper>
+            <Buttons
+              buttonText="Book Now"
+              fontSize={15}
+              paddinglr={windowWidth >= 1024 ? 48 : 16}
+              width={windowWidth >= 744 ? 70 : windowWidth >= 448 ? 100 : 95}
+              className="class-button"
+            />
+          </StyledButtonWrapper>
+        </StyledTextButton>
+      </StyledCard>
+    </>
+  );
+};
```

Comment:
Why couldn't you use a unique component instead of creating different cards which have the same structure?